SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -72,6 +75,17 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 				}
 
 				bWroteSomething = true;
+#if USE_NETWORK_PROFILER
+				/**
+				 *  a good proxy. Reasons for why it might not be fully accurate:
```

Comment:
did you miss a line in your CTL-C CTL-V?

Reply:
No, but I'll update it.